requirements-external.txt

```diff
@@ -6,5 +6,5 @@ git+https://github.com/opendatahub-io/oauthenticator@42c4790a2db77ff4259a74e15ed
 # jupyterhub-singleuser-profiles/jupyterhub_singleuser_profiles/ui/package.json
 # jupyterhub-singleuser-profiles/jupyterhub_singleuser_profiles/ui/package-lock.json
 #
-git+https://github.com/red-hat-data-services/jupyterhub-singleuser-profiles@4858fb981d1947ead00686b0e4e359282e5abdfa#egg=jupyterhub-singleuser-profiles
+git+https://github.com/maroroman/jupyterhub-singleuser-profiles@4f7c496539e1c4060986946d95c703b4a29e01cc#egg=jupyterhub-singleuser-profiles
```

Comment:
This is referring to the wrong repo

Reply:
/hold

Officially marking this with `/hold` to make that this line will be updated after https://github.com/red-hat-data-services/jupyterhub-singleuser-profiles/pull/60